src/main/resources/assets/identity/lang/zh_cn.json

```diff
@@ -1,13 +1,25 @@
 {
-  "identity.unlock_entity": "已解锁%s种化身",
-  "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃...",
-  "identity.grant_success": "成功将化身%s给予%s。",
-  "identity.revoke_success": "成功删除%s化身(%s身上的化身)。",
-  "identity.already_has": "%s已经拥有%s化身！",
-  "identity.does_not_have": "%s没有%s化身！",
-  "identity.equip_success": "将%s的化身设置为%s。",
-  "identity.unequip_success": "已删除%s化身。",
-  "identity.test_positive": "%s化身为%s。",
-  "identity.test_failed": "%s没有化身为%s",
-  "identity.menu_hint": "杀死任何生物以获得化身..."
-}
+    "identity.unlock_entity": "已解锁%s的化身",
+    "identity.revoke_entity": "你化身为%s的记忆已逐渐消逝......",
+    "identity.grant_success": "已成功将%s的化身转给%s。",
+    "identity.revoke_success": "已成功将%s的化身转回给%s。",
+    "identity.already_has": "%s已经拥有%s的化身了！",
+    "identity.does_not_have": "%s还没有%s的化身！",
+    "identity.equip_success": "%s的化身已装备给%s。",
+    "identity.unequip_success": "已移除%s的化身。",
+    "identity.test_positive": "%s已经伪装成%s了。",
+    "identity.test_failed": "%s还没被伪装成%s。",
+    "identity.menu_hint": "杀死任意一种生物来在你的世界里获得化身。",
+    "identity.help": "打开《化身》模组帮助按钮",
+    "identity.help.welcome": "欢迎使用《化身》，这是模组《变形（Morph）》的1.16+ Fabric版本，由Draylar编写，由Godzmoline汉化。",
```

Comment:
I would like to have this line only reference my name (because I am the primary author/contact). To still properly credit contributors to the project, I just pushed a new update with a "Credits" category in this menu:

![image](https://user-images.githubusercontent.com/17728338/104850641-d3c28e80-58b5-11eb-9f1d-1523caf94c63.png)

The translation keys are as follows:
```java
  "identity.help.credits_label": "Credits",
  "identity.help.credits_general": "General: Pyrofab, AllanChain, James103, lawleagle",
  "identity.help.credits_translators": "Translators: qsefthuopq"
```
Please add your name to this instead (I will add your name to the English version as well).

Reply:
That's OK, I don't mind.